### PR TITLE
ansible-test sanity

### PR DESCRIPTION
**Pull request by gundalow**
Use ansible-test to validate Galaxy roles

### CODE REVIEW

File: zuul.d/jobs.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- job:
+    name: ansible-test-sanity
+    description: ansible-test sanity
+    required-projects:
+      - name: ansible/ansible
+        override-checkout: stable-2.5
+    vars:
+      python_version: 2.7
```
Comment (TristanCacqueray) [inline zuul.d/jobs.yml]: maybe use: python_version: python2.7

File: zuul.d/jobs.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+- job:
```
Comment (pabelanger) [inline zuul.d/jobs.yml]: You need to specific the playbook to run:
  run: playbooks/ansible-test-sanity/run.yaml

https://docs.openstack.org/infra/zuul/user/config.html#attr-job.run

File: zuul.d/jobs.yml (hunk 1 above)
Comment (gundalow) [reply zuul.d/jobs.yml]: In this case I want just the version so I can do
`ansible-test [sanity,network-integration] --python 2.7 ...`

File: zuul.d/jobs.yml (hunk 2 above)
Comment (gundalow) [reply zuul.d/jobs.yml]: Added `run: playbooks/ansible-test-sanity/run.yml`

**Comment by gundalow** [comment]
Deleted and moved to https://github.com/ansible-network/ansible-zuul-jobs/pull/4

**Pull request closed without merging**